backend/build.gradle.kts

```
@@ -23,21 +23,31 @@ dependencies {
     annotationProcessor("io.micronaut.security:micronaut-security-annotations")
     annotationProcessor("io.micronaut.data:micronaut-data-processor")
     annotationProcessor("io.micronaut:micronaut-inject-java")
+    annotationProcessor("io.micronaut.validation:micronaut-validation-processor")
     // When.MAYBE warning fix
     annotationProcessor("com.google.code.findbugs:jsr305:3.0.2")
     implementation("ch.qos.logback:logback-classic:1.4.11")
+    implementation("org.slf4j:slf4j-api:2.0.7")
     implementation("com.google.code.findbugs:jsr305:3.0.2")
     implementation("io.micronaut.security:micronaut-security-jwt")
     implementation("io.micronaut.reactor:micronaut-reactor")
     implementation("io.micronaut:micronaut-runtime")
     implementation("io.micronaut.data:micronaut-data-jdbc")
+    // https://github.com/flyway/flyway/issues/3651
+    // 6.0.0-M2 uses 9.16.1 which is before bug introduced in flyway
     implementation("io.micronaut.flyway:micronaut-flyway")
+    implementation("org.flywaydb:flyway-core") {
+        version {
+            strictly("9.16.1")
+        }
+    }
```

Updating the comment to match the new state
```suggestion
    implementation("io.micronaut.flyway:micronaut-flyway")
    // https://github.com/flyway/flyway/issues/3651
    implementation("org.flywaydb:flyway-core") {
        version {
            strictly("9.16.1")
        }
    }
```